src/attributes.js

```diff
@@ -122,19 +123,20 @@ var defaults = {
  * A publicly mutable object to provide custom mutators for attributes.
  * @const {!Object<string, function(!Element, string, *)>}
  */
-var mutators = {
+var attributes = {
   // Special generic mutator that's called for any attribute that does not
   // have a specific mutator.
   [symbols.all]: applyAttributeTyped,
 
   // Special case the style attribute
-  'style': applyStyle
+  'style': applyStyle,
+
+  [symbols.defaults]: defaults
```

Comment:
Looking at it again, I'm not sure we really need `defaults` at all. Both `applyProp` and `applyAttr` are very simple, and they're the only two I ever imagine using.